feature/foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt

```diff
@@ -182,7 +182,7 @@ internal fun ForYouScreen(
             onTopicClick = onTopicClick,
         )
 
-        item(span = { GridItemSpan(maxLineSpan) }) {
+        item(span = { GridItemSpan(maxLineSpan) }, contentType = { "bottomSpacing" }) {
```

Comment:
Nit: Why is this `item` `contentType` a lambda, but the `contentType` in the `item` later in this file just a `String`? Not sure if it matters at all, but seems inconsistent.

Reply:
This is what the API requires.

When you use `items` (plural), it gives you lambda, because you can use the item inside the lambda to decide on `contentType`.

For `item` (singular)`, it takes directly the string, because you know what item you pass.

Reply:
Possibly because `LazyGridScope.items` (with an s) has a `contentType` lambda rather than a simple type. There doesn't seem to be any need for it here.

Reply:
@mlykotom Posted my previous comment before I saw that you'd replied. You're calling `item` not `items` so should use `contentType = "bottomSpacing"`.

Reply:
Ah, good catch. Fixed now.